manifests/config/backend.pp

```diff
@@ -51,21 +55,23 @@
 # Alex Schultz <aschultz@next-development.com>
 #
 define pgpool::config::backend (
-  $ensure         = present,
-  $id             = 0,
-  $hostname       = 'localhost',
-  $port           = 5432,
-  $weight         = 1,
-  $data_directory = '/var/lib/pgsql/9.3/data',
-  $flag           = 'ALLOW_TO_FAILOVER',
+  $ensure           = present,
+  $id               = 0,
+  $hostname         = 'localhost',
+  $port             = 5432,
+  $application_name = 'localhost',
```

Comment:
I wonder if we should use $title here instead.

https://puppet.com/docs/puppet/6.17/lang_defined_types.html#dollartitle-and-dollarname

Since it's configurable, i'll just merge as is but $title might be a better default

Reply:
You are right, indeed I also wondered about that — but then, a user might also expect that `$hostname` would be `$title`, and I was unsure whether it would be a good choice to have both `$hostname` and `$application_name` default to `$title`, which would then not only introduce `$application_name`, but also change existing functionality. 

What do you think about this? If you want, I could do that in a separate PR.

Reply:
I would switch it out to $title if you have time.

Reply:
Ok, done in #26 , have tested that just now on our pgpool instance (it's still not in production, so testing is for free ;-) ).